docs/source/pages/BW1094.rst

```diff
@@ -10,8 +10,8 @@ DepthAI RPi HAT Baseboard
 Overview
 ********
 
-The DepthAI RPi HAT Baseboard (BW1094) baseboard has three FFC interfaces which allow for two :ref:`OAK-FFC-OV9282 <bg0250tg>` camera modules (stereo pair) and one 
-:ref:`OAK-FFC-IMX378 <bg0249>` RGB camera module. The GPIO interface from the Raspberry Pi Model B variants is passed through the BW1094 with 
+The **OAK-FFC RPi HAT** (BW1094) baseboard has three FFC interfaces which allow for two :ref:`OAK-FFC-OV9282 <bg0250tg>` camera modules (stereo pair) and one
```

Comment:
Does it support other CBA we've recently added (-FF, -W)? If yes, can you mentioned them?

Reply:
Will address in next PR